app/src/main/java/ch/admin/bag/dp3t/MainApplication.java

```diff
@@ -130,6 +131,16 @@ public void onReceive(Context context, Intent intent) {
 	private static void createNewContactNotification(Context context, int contactId) {
```

Comment:
General suggestion: Extracting the notification management to a separate class (`ContactNotificationManager` or alike), to avoid doing everything in the Application class and inflating it.